components/badge/badge.html

```diff
@@ -50,7 +50,30 @@
 
     <!-- || Icons with a Badge ends -->
 
+   <!-- || Tag Badge start -->
 
+   <div class="component-container">
+    <div class="badge-container tag tag-primary">
```

Comment:
Maybe it's intentionally put so but just a suggestion that add the `tag-round-border` class to all the badges so it looks uniform.

Reply:
Great suggesstion, The idea for keeping it that way, so user can see both changes side by side and decide for right one to use. Thanks, I will look into this.